Readme update for docker publish for gateway config (#232)
mosaicdao/mosaic-chains: docker/readme.md

    ```
5. Move the mosaic config file in `configs` directory.

> 6. Move the gateway configs file in respective folder. Final folder structure for gateway config should look like below:

Reviewer: ```suggestion
6. Move the gateway config for a token to a folder having below folder structure:
```

Author: Changes done!
